api/src/main/java/pl/kolendateam/dadcard/feats/dto/FeatsDTO.java

```diff
@@ -12,14 +13,17 @@ public class FeatsDTO {
     public int id;
     public String featName;
     public FeatsTypeEnum featsType;
-    public String prerequisite;
+    public PrerequisiteDTO prerequisite;
     public String description;
 
     public FeatsDTO(Feats feats){
         this.id = feats.getId();
         this.featName = feats.getFeatName();
         this.featsType = feats.getFeatsType();
-        this.prerequisite = feats.getPrerequisite();
+        if(feats.getPrerequisite() == null){
+            this.prerequisite = null;
+        } else {
+        this.prerequisite = MapperPrerequisiteDTO.toPrerequisiteDTO(feats.getPrerequisite());}
```

Comment:
zbij te } nizej bo przez chwile mysla…em ze "this.description = feats.getDescription();" tez jest w else